flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -32,30 +32,24 @@ if [ -z "$FLINK_DIR" ] ; then
     exit 1
 fi
 
+if [ -z "$FLINK_LOG_DIR" ] ; then
+    export FLINK_LOG_DIR="$FLINK_DIR/logs"
```

Comment:
I think this change breaks the e2e setup on my machine (MacOS) because the directory does not exist.